ethicml/vision/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Module containing functionality for vision tasks such as face recognition."""
+from .data import *
```

Comment:
I've come to accept the use of wildcards (after long discussions with Myles) under the following conditions:

- it's in an `__init__.py`
- the file that we're importing from defines `__all__ = [...]`

Under these circumstances, it basically is equivalent to an explicit import because the `*` will import exactly those things that are defined in `__all__`, so you still have full explicit control over what is imported.

Now you might ask, if it is equivalent then why use `*` + `__all__` over the import with the list? The advantage of the  `*` + `__all__` method is that when you add or remove or rename a function or class then the change is *local*. You do not have to go and edit the `__init__.py` which is always annoying and is often forgotten.

Reply:
Other advantages of `*` + `__all__`:

- leads to fewer merge conflicts
- forces you to think about the API design while your still in the file as opposed to later when you're in a different mode of thinking
- it tells the IDE which things are ok to suggest to import from a file (namely the things in `__all__`), so it won't suggest importing `pandas` from a file that happens to import `pandas` as well

Reply:
That is a robust comment. Thank you